flocker/ca/_ca.py

```diff
@@ -332,6 +355,36 @@ class UserCredential(PRecord):
     username = field(mandatory=True, type=unicode)
 
     @classmethod
+    def from_files(cls, certificate_path, key_path):
```

Comment:
:question: I'll look into this more but first impression is why is this better than `from_path` - the docstring seems to say that it is because this loads "a user certificate and keypair" and `from_path` loads `a user certificate`. If that is the difference then either something should be moved or something should be renamed.